_posts/2022-06-22-#23418.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: pr
+date: 2022-06-22
+title: "Fix signed integer overflow in prioritisetransaction RPC"
+pr: 23418
+authors: [MarcoFalke]
+components: ["rpc/rest/zmq", "Mempool"]
+host: MarcoFalke
+status: upcoming
+commit:
+---
+
+
+## Notes
+
+* In C++, unsigned integer overflow is well defined and often used by design.
+  However, signed integer overflow is in a strict sense, undefined behavior
+  (UB).
```

Comment:
Since the abbreviation isn't used anywhere, I'd just leave it out?
_edit: noticed that you use UB in the PR description. Would still suggest to leave it out here, and instead update the PR description to describe the full acronym of UB there._
```suggestion
  However, signed integer overflow is in a strict sense, undefined behavior.
```